tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -307,7 +307,7 @@ async def ensure_wallet_sync() -> None:
     )
     assert len(txs) == 1
     assert txs[0].spend_bundle is not None
-    await wallet_node_0.wallet_state_manager.add_pending_transaction(txs[0])
```

Comment:
this seems to be a recurring patter. I'll just point out all the places I find that look suspicious